draft-ietf-dmarc-dmarcbis.md

```diff
@@ -83,41 +82,40 @@ authenticated for that message. Authentication does not carry an
 explicit or implicit value assertion about that message or about
 the Domain Owner. Furthermore, a mail-receiving organization that performs
 DMARC verification can choose to honor the Domain Owner's requested
-message handling for authentication failures, but it is under no
-obligation to do so; it might choose different actions entirely.
+message handling for authentication failures, but it is not obligated to do so; it might choose different actions entirely.
 
 For a mail-receiving organization supporting DMARC, a message that
-passes verification is part of a message stream that is reliably
+passes verification is part of a message stream reliably
 associated with the RFC5322.From field Domain Owner. Therefore,
 reputation assessment of that stream by the mail-receiving organization
-can assume that use of that domain
-in the RFC5322.From field is authorized.  A message that fails this verification
+can assume the use of that domain
+in the RFC5322.From field is authorized. A message that fails this verification
 is not necessarily associated with the Domain Owner's domain and its
 reputation.
 
-DMARC policy records can also cover non-existent sub-domains, below the
+DMARC policy records can also cover non-existent sub-domains below the
 "Organizational Domain", as well as domains at the top of the name hierarchy,
 controlled by Public Suffix Operators (PSOs).
 
 DMARC, in the associated [@!I-D.ietf-dmarc-aggregate-reporting] and [@!I-D.ietf-dmarc-failure-reporting] ; 
-documents, also specifies a reporting framework. Using it, a mail-receiving
+documents also specify a reporting framework. Using it, a mail-receiving
```

Comment:
I don't agree with this change. The subject here is DMARC, not the documents, and therefore the use of "specifies" was correct, i.e., "DMARC also specifies a reporting framework..."